src/docs/release/breaking-changes/platform-views-using-html-slots-web.md

```diff
@@ -0,0 +1,196 @@
+---
+title: Using HTML slots to render platform views in the web
+description: iframes in Flutter web used to reload, because of the way some DOM operations were made. A change in the way Flutter web apps render Platform Views makes them much more stable (preventing iframe reloads, and other problems with video tags or forms potentially losing their state).
+---
+
+## Summary
+
+Flutter now renders Platform Views using [slot elements][] inside of a single,
+app-wide [Shadow Root][]. Slot elements can be added/removed/moved around the
+Shadow DOM without affecting the underlying slotted content (which is rendered
+in a constant location)
+
+## Context
+
+The Flutter framework frequently tweaks its Render Tree to optimize the paint
```

Comment:
I'm not sure if "render tree" is worth capitalizing.

Reply:
Capitalization Of Things, The Bane Of My Existence :P

Reply:
(lower-cased :P)